qpmodel/PhysicalNode.cs

```diff
@@ -1697,17 +1707,33 @@ public override void Exec(Action<Row> callback)
                 {
                     context.code_ += $@"
                     {_physic_}.nrows_++;
+                    if ({_physic_}.aggProfiling_ != null)
+                    {{
+                        {_physic_}.aggProfiling_.nrows_++;
+                    }}
                     var r{_} = r{child_()._};";
                     callback(null);
                 }
                 else
                 {
                     nrows_++;
+                    if (aggProfiling_ != null)
+                    {
+                        aggProfiling_.nrows_++;
+                    }
                     callback(l);
                 }
             });
         }
 
+        // assign aggregation profiling to cloned profiling.
+        public override PhysicNode Clone()
```

Comment:
Query plan under Gather with actual "actual rows = 0" because of all Profilings under Gather are cloned. So we should keep reference of original Profiling in all cloned profilings.

All expected results are updated.